SKPhotoBrowser/SKZoomingScrollView.swift

```diff
@@ -225,7 +225,9 @@ public class SKZoomingScrollView:UIScrollView, UIScrollViewDelegate, SKDetecting
     }
     
     func handleDoubleTap(view: UIView, touch: UITouch) {
-        // nothing to do
+        var point = self.convertPoint(touch.locationInView(view), toView: self.photoImageView)
+        point.y = 0
```

Comment:
Could you tell me why you insert point.y = 0.
It's not the accurate point for use tap I think